deploy/5_safe.js

```diff
@@ -0,0 +1,46 @@
+module.exports = async (hre) => {
+    // Based on https://github.com/gnosis/safe-tasks/blob/master/src/creation.ts
+
+    // Get the signers (default, pre-funded accounts)
+    const accounts = await hre.ethers.getSigners();
+
+    safe_factory_address = "0x5FbDB2315678afecb367f032d93F642f64180aa3";
+    safe_address = "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f"; // account 8
+    account_address = "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720"; // account 9
+
+    // Get safe factory instance
+    safe_factory_contract = await hre.ethers.getContractAt("GnosisSafeProxyFactory", safe_factory_address);
+
+    // Get safe? instance
+    // Can we use this 'safe_address' here? It is not deployed yet, so I don't think this will work. They call this singleton/master,
+    // but in the oracle code this seems the safe to me.
+    safe_contract = await hre.ethers.getContractAt("./third_party/safe-contracts/contracts/GnosisSafe.sol:GnosisSafe", safe_address); // Doesn't work for GnosisSafeL2 yet
```

Comment:
I don't think we can do this. If the safe is not deployed yet, how can we use that address. This might have to do with how they calculate an address later. But in the python code for the oracle seems that we pass the safe address here.

Reply:
For reference: https://github.com/gnosis/safe-tasks/blob/master/src/creation.ts